masterlist.yaml

```
@@ -23208,6 +23208,11 @@ plugins:
     after:
       - 'Qw_ACE_CRF Patch.esp'
       - 'Qw_WACCF_CRF Patch.esp'
+  - name: 'DBM_Requiem_Patch.esp'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Requiem - Legacy of the Dragonborn patch' ]
+        condition: 'many("(DBM_Requiem_Patch|Requiem - Legacy of the Dragonborn)\.esp")'
```

`many()` is really only useful for regex plugins. The following is equivalent & simpler in this case due to the fact that it's implicitly assumed that `DBM_Requiem_Patch.esp` is present if the message is being displayed:

```suggestion
        condition: 'file("Requiem - Legacy of the Dragonborn.esp")'
```